modules/FindOVR.cmake

```diff
@@ -39,28 +39,61 @@
 #   DEALINGS IN THE SOFTWARE.
 #
 
-set(LIBOVR_ROOT ${CMAKE_INSTALL_PREFIX})
-
 if(NOT OVR_SDK_ROOT)
-    message(WARNING "OVR_SDK_ROOT is not set. Will try to find headers and library in ${CMAKE_INSTALL_PREFIX}." )
-else()
-    set(LIBOVR_ROOT ${OVR_SDK_ROOT}/LibOVR)
+   find_path(OVR_SDK_ROOT OculusSDK)
+endif()
+
+if(OVR_SDK_ROOT)
+    set(OVR_SDK_ROOT "${OVR_SDK_ROOT}/OculusSDK")
+
+    mark_as_advanced(OVR_SDK_ROOT)
```

Comment:
Only variables with explicit `CACHE` setup or variables coming from `find_*()` commands need to be marked as advanced. All other variables don't go into the cache at all.